Tests/fixtures/tv-3.yml

```diff
@@ -3081,3 +3081,21 @@
     model: ""
   os_family: Windows
   browser_family: Chrome
+-
+  user_agent: TV Bro/1.0 Mozilla/5.0 (Linux; Android 9; TriaPlayBox Build/PI; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/119.0.6045.163 Safari/537.36
+  os:
+    name: Android
+    version: "9"
+    platform: ""
+  client:
+    type: browser
+    name: TV Bro
+    version: "1.0"
+    engine: Blink
+    engine_version: 119.0.6045.163
+  device:
+    type: tv
+    brand: TriaPlay
```

Comment:
https://storage.googleapis.com/play_public/supported_devices.html
https://staticweb.tvbanywhere.com/certified-android-TV-list.pdf
https://support.fubo.tv/hc/en-us/articles/115002183228-What-devices-can-I-watch-Fubo-on
https://developers.google.com/zero-touch/resources/manufacturer-names

| Manufacture |  TV Provider |
| --- | --- |
Skyworth   | GO3
Geniatech | TriaPlay

I trust the certified list more

Reply:
should not the brand be `Geniatech` then, and `TriaPlay` - model?

Reply:
We already have other brands like this. `Orange`, `Vodafone`, `Turkcell`, `Verizon`, `TCL A2 (TracFone A507DL)` for example. It's better to stick with Google Play naming where possible.

@sanchezzzhak probably is best to add a comment to the regex for future detections, so we know is a `Geniatech`.

Reply:
@Simbiat not always, since major players can produce products for a specific brand or trade market
(not every brand has its own capabilities to create a full-fledged production)

Reply:
My problem is more with the fact, that I can't find any evidence of a device or brand existing. Orange, Vodafone and other ISPs - they have websites. TriaPlay seems to be mentioned only in such lists, as mentioned above.